yapapi/network.py

```diff
@@ -261,6 +261,21 @@ async def add_node(self, node_id: str, ip: Optional[str] = None) -> Node:
 
         return node
 
+    async def refresh_nodes(self):
+        for node in self._nodes.values():
+            try:
+                await self._net_api.add_node(self.network_id, node.node_id, node.ip)
+            except ApiException as e:
```

Comment:
Please add some comment here.
Like "this is not expected with latest yagna, but used to happen with older versions, so we guard against it"